src/main/java/jnr/posix/LinuxPOSIX.java

```diff
@@ -44,16 +54,16 @@ public FileStat allocateStat() {
             if ("aarch64".equals(Platform.ARCH)) {
                 return new LinuxFileStatAARCH64(this);
             } else if ("sparcv9".equals(Platform.ARCH)) {
-		return new LinuxFileStatSPARCV9(this);
-	    } else if ("loongarch64".equals(Platform.ARCH)) {
-		return new LinuxFileStatLOONGARCH64(this);
-	    } else {
-		if (Platform.ARCH.contains("mips64")) {
-		    return new LinuxFileStatMIPS64(this);
-		}
+                return new LinuxFileStatSPARCV9(this);
+            } else if ("loongarch64".equals(Platform.ARCH)) {
+                return new LinuxFileStatLOONGARCH64(this);
+            } else {
+                if (Platform.ARCH.contains("mips64")) {
+                    return new LinuxFileStatMIPS64(this);
+                }
                 return new LinuxFileStat64(this);
-	    }
-	}
```

Comment:
We appreciate whitespace cleanup but it should be done as a separate commit to avoid confusing history.